docs/source/aws/contribute.rst

```diff
@@ -0,0 +1,169 @@
+.. _aws_contribute:
+
+Developer Guide
+=================
+
+Cloud Custodian is a Python application and supports Python 2 and 3 on Linux and Windows. It is recommended 
+using Python 3.6 or higher.
+
+Run the following commands in the root directory after cloning Cloud Custodian:
+
+.. code-block:: bash
+
+    $ make install
+    $ source bin/activate
+
+This creates a virtual env in your enlistment and installs all packages as editable.
+
+Now you may run ``python3 c7n/cli.py`` with any flags in order to directly test changes to the source files.  For example, 
```

Comment:
? if you have a developer install per make install, and the virtualenv activate, executing the source directly versus using the cli entrypoint is superflous, yes, its possible, but people should just use the cli `custodian` entrypoint since its the same workflow and command across usages instead of introducing additional one off extraneous mechanisms into the docs.